packages/vue-client/src/components/boards/MulticolorGridBoard.vue

```diff
@@ -15,7 +15,7 @@ import { positionsGetter } from "./board_utils";
 const props = defineProps<{
   board: (MulticolorStone | null)[][];
   background_color?: string;
-  config: { height: number; width: number };
+  config: { width: number; height: number };
```

Comment:
Perhaps change this property name to dimensions since it's no longer representative of the game config.

Reply:
changed props name to board_dimensions